auto_create.go

```diff
@@ -52,6 +52,12 @@ const (
 	defaultAutoCreateAppVersion = "1.0.0"
 	defaultAutoCreateAppLabel   = "app"
 
+	// AppPartOf ...
```

Comment:
Nit:  seems unnecessary to have two constants with same value.

Reply:
done